lib/modules/forgot_password/forgot_password_presenter.dart

```diff
@@ -13,4 +21,30 @@ class ForgotPasswordPresenterImpl extends ForgotPasswordPresenter
 
   @override
   final alertDialogDidClose = BehaviorSubject<void>();
+
+  @override
+  final passwordDidReset = BehaviorSubject<void>();
+
+  @override
+  final passwordDidFailToReset = BehaviorSubject<Exception>();
```

Comment:
as I mentioned in the prev PR, the point of enableForgotPassword button should be in this PR (Integrate) task, and imo, it should stay somewhere here no?

Reply:
@sleepylee sorry, don't get your point here 😬

Reply:
before, we mentioned control extending the state management in View, and also, you do the state binding in a [UI] PRs, making it very big to follow.

Reply:
@sleepylee ah yea, I realized this after did it, so I changed it in my next implements